### PR TITLE
[minor] Added HMAC to COS role

### PR DESCRIPTION
**Pull request by Vishal-patel-7**
As a part of https://jsw.ibm.com/browse/MAXMF-455 we added new environment variable USE_HMAC to create COS with HMAC support. There is PR https://github.com/ibm-mas/ansible-devops/pull/1407 in ansible-devops regarding this.

### CODE REVIEW

**Review by whitfiea — CHANGES_REQUESTED** [review]
The ansible-devops role now allows the config scope to be set and the use_hmac to be set, which is all fine. But that means that for an env that has assist, mvi and manage installed we need the following set:
- objectstorage-cfg at system scope with use_hmac as false
- objectstorage-cfg at app/workspace(for manage) scope with use_hmac as true

I don't think the changes here allow the scope to be set or for more than one config/dependency to be created

**Comment by Vishal-patel-7** [comment]
> The ansible-devops role now allows the config scope to be set and the use_hmac to be set, which is all fine. But that means that for an env that has assist, mvi and manage installed we need the following set:
> 
> * objectstorage-cfg at system scope with use_hmac as false
> * objectstorage-cfg at app/workspace(for manage) scope with use_hmac as true
> 
> I don't think the changes here allow the scope to be set or for more than one config/dependency to be created

I don't know how to handle this but the aim for the changes are to give new environment variable support for ansible-devops role. @chiragcsatani can you confirm what we need to do ? This is to fix fvt issue for manage but for assist & mvi we may have to revisit this code and ansible-devops role also.

File: tekton/src/pipelines/gitops/gitops-mas-deps.yml.j2 [hunk 1]
```diff
@@ -214,6 +218,8 @@ spec:
           value: $(params.ibmcloud_resourcegroup)
         - name: ibmcloud_apikey
           value: $(params.ibmcloud_apikey)
+        - name: use_hmac
+          value: $(params.use_hmac
```
Comment (whitfiea) [inline tekton/src/pipelines/gitops/gitops-mas-deps.yml.j2]: missing bracket

**Review by whitfiea — CHANGES_REQUESTED** [review]

File: tekton/src/pipelines/gitops/gitops-mas-deps.yml.j2 (hunk 1 above)
Comment (Vishal-patel-7) [reply tekton/src/pipelines/gitops/gitops-mas-deps.yml.j2]: Corrected..

**Review by whitfiea — APPROVED** [review]

**Pull request merged**